src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -72,11 +55,11 @@ trait Backend {
   }
 
   /**
-   * Return a possibly altered copy of inputs reflecting any localization of input file paths that might have
-   * been performed for this `Backend` implementation.
-   */
+    * Return a possibly altered copy of inputs reflecting any localization of input file paths that might have
+    * been performed for this `Backend` implementation.
+    */
   def adjustInputPaths(callKey: CallKey,
-                       runtimeAttributes: CromwellRuntimeAttributes,
+                       runtimeAttributes: RuntimeAttributes,
```

Comment:
The CromwellRuntimeAttributes was intentional - but this goes back to the removal of the backend specific knowledge which we agreed would be put back into place